src/components/MinimalCard.js

```diff
@@ -19,9 +19,9 @@ export function MinimalCard(props) {
   return (
     <>
       <button
-        className={`nfd-ecommerce-card ${
-          taskCompleted ? "nfd-ecommerce-taskCompleted" : ""
-        }`}
+        className={`nfd-ecommerce-card
+         ${taskCompleted ? "nfd-ecommerce-taskCompleted" : ""} 
+         ${disableCard ? " nfd-ecommerce-disable" : ""}`}
```

Comment:
Instead of adding a new css class for this, we can instead set the `disabled` prop on the `button` and then use `button.nfd-ecommmerce-card:disabled` pseudo-class to style it.